src/GraphQL.Tests/StarWars/StarWarsSubFieldsTest.cs

```diff
@@ -108,7 +108,7 @@ public void subfields_is_not_null_for_single_InterfaceGraphType()
                ctx.SubFields.ShouldNotBeNull();
                ctx.SubFields.Keys.ShouldContain("id");
                ctx.SubFields.Keys.ShouldContain("friends");
-               return null;
+               return Task.FromResult<object>(null);
```

Comment:
I think that this PR may allow to cut some APIs from `Field/FieldAsync` family. I'm not sure.

Reply:
Anyway I would like to switch all `Field/FieldAsync` overloads across code base to prefer `ValueTask` over `Task`. One could just use a simple conversion like `return new ValueTask<int>(my_Task_Of_Int_From_Some_External_Source)` or just `return await my_Task_Of_Int_From_Some_External_Source` to make transition from `Task` to `ValueTask`.